### PR TITLE
Eagerly dialing masquerades in parallel, using shorter timeout

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

**Comment by oxtoacart** [comment]
@myleshorton I don't have a working cross-compilation environment for the release branch.  If you do, would you mind building a linux executable and trying that on the Iran VPS?

File: src/github.com/getlantern/flashlight/client/fronted.go [hunk 1]
```diff
@@ -1,17 +1,5 @@
 package client
 
-import (
```
Comment (oxtoacart) [inline src/github.com/getlantern/flashlight/client/fronted.go]: I took out a bunch of unused fronted stuff while I was in here, which also meant taking out the unused server functionality in flashlight.

**Comment by myleshorton** [comment]
I'm still getting `too many open files` sometimes locally on this branch @oxtoacart. It also looks like vetting all masquerades upfront often leads to issues with the startup sequence -- I so far haven't been able to get Lantern to successfully proxy.

**Comment by myleshorton** [comment]
This is working great on the field @oxtoacart! Merging and building!

**Pull request merged**